frontend/ui/elements/screen_rotation_menu_table.lua

```diff
@@ -81,7 +81,7 @@ When checked, in addition to the obvious, setting a default rotation below will
                     if G_reader_settings:nilOrFalse("lock_rotation") then
                         UIManager:show(InfoMessage:new{
                             text = _("Can't set a default FM rotation when 'Keep file browser rotation' is disabled!"),
-                            timeout = 2,
+                            timeout = 4,
```

Comment:
I would even remove the timeout: it's an InfoMessage, it can be dismissed - and I might need more than 4s to process it and understand what I should do :)
Also, the `text = _("Can't set ...")` string looks like it's the same duplicated 4 times: might be better put in a local variable above, so we never risk to update one and forget the 3 others.